apsuite/commissioning_scripts/apply_strengths.py

```diff
@@ -306,7 +310,7 @@ def apply_model_strengths(
                 magidx = fam['QN']['index']
                 # NOTE: This is incorrect for magnets with more than one
```

Comment:
@murilobalves, I think this `NOTE` would be better placed inside the method `_get_idx`, where the algorithm is really placed.

Reply:
Indeed. Done!